packages/suite/src/components/suite/FormattedCryptoAmount/index.tsx

```diff
@@ -9,29 +15,72 @@ const Value = styled.span`
 `;
 
 const Symbol = styled.span`
-    text-transform: uppercase;
     word-break: initial;
 `;
 
-interface Props {
-    value: React.ReactNode;
-    symbol?: string;
+interface FormattedCryptoAmountProps {
+    value: string | number | undefined;
+    symbol: string | undefined;
+    signValue?: SignValue;
     disableHiddenPlaceholder?: boolean;
+    isRawString?: boolean;
+    'data-test'?: string;
     className?: string;
 }
 
-const FormattedCryptoAmount = ({ value, symbol, disableHiddenPlaceholder, className }: Props) => {
+export const FormattedCryptoAmount = ({
```

Comment:
Incomming and Outgoing amounts overflows instead of elipsis

this branch:
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/3729633/180261559-3ae9749b-3eb3-42bd-8743-839478188325.png">

current develop:
<img width="1510" alt="image" src="https://user-images.githubusercontent.com/3729633/180261574-f8077b7d-8ac1-4c79-b047-9f6a85739fe1.png">

Reply:
cfb2e6a6ffdf6190073f64e819e1ca1427d6fb8c

Also using  the `testnet` property there.